probes/commitsInLast90Days/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: commitsInLast90Days
+short: Check whether the project has at least one commit per week over the last 90 days.
+motivation: >
+  A project which is not active might not be patched, have its dependencies patched, or be actively tested and used. However, a lack of active maintenance is not necessarily always a problem. Some software, especially smaller utility functions, does not normally need to be maintained. For example, a library that determines if an integer is even would not normally need maintenance unless an underlying implementation language definition changed. A lack of active maintenance should signal that potential users should investigate further to judge the situation.
+implementation: >
+  The implementation checks the number of commits made in the last 90 days by any user type.
+outcome:
+  - If the project has commits from the last 90 days, the probe returns as many OutcomePositive as the project has commits.
+  - If the project does not have commits in the last 90 days, the probe returns a single OutcomeNegative.
+remediation:
+  effort: Low
+  text:
+    - Make contributions to the project.
```

Comment:
More of a philosophical question, but some projects are "done". I'm curious if a remediation really makes sense here @laurentsimon

Reply:
by "done" do you mean we can't undo the commits?

Reply:
I'm saying if the project/library is feature complete, should we really be recommending contributions? If nothing is broken, no vulns are present, no dependencies need updating, do people really need to make commits?

Reply:
makes sense now. Agreed on the comment. What could we say instead?

Reply:
similar to issue activity probe. name could probably drop 90 days.

Remediation text could be more neutral. If project is feature complete / doesn't need maintenance, don't worry about this, it's just a heuristic.

Reply:
Added in https://github.com/ossf/scorecard/pull/3507/commits/dea9bd13e19f8c8b78ca60fab1d953b75da8a0b4